src/features/hooks/use-residence-list/index.tsx

```diff
@@ -1,29 +1,46 @@
 import { useEffect, useMemo } from 'react'
-import useWS from '../../../components/hooks/useWS'
+import useWS from 'components/hooks/useWS'
 
 export type ResidenceType = {
     text: string
     value: string
 }
 
-const formatResidenceList = (residences) => {
+const formatResidenceList = ({
+    residences,
+    restricted,
+}: {
+    residences: any[]
+    restricted: string[]
+}) => {
```

Comment:
Can we specify/enumerate the type of `residences` and `restricted`?

Reply:
it's for about 250 countries here so no